build_projects/shared-build-targets-utils/Publishing/AzurePublisher.cs

```diff
@@ -55,7 +55,7 @@ public void PublishArchive(string archiveFile, string channel, string version)
         public void PublishFile(string blob, string file)
         {
             CloudBlockBlob blockBlob = _blobContainer.GetBlockBlobReference(blob);
-            blockBlob.UploadFromFileAsync(file, FileMode.Open).Wait();
+            blockBlob.UploadFromFileAsync(file).Wait();
```

Comment:
Are the two calls equivalent? Also, do you think it is worth doing a private official build with this?